docs/src/pages/components/slider-styled/CustomizedSlider.js

```diff
@@ -1,19 +1,21 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Slider from '@material-ui/lab/SliderStyled';
-import styled from '@emotion/styled';
 import Typography from '@material-ui/core/Typography';
 import Tooltip from '@material-ui/core/Tooltip';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: `calc(300px + ${theme.spacing(6)})`,
-  },
-  margin: {
-    height: theme.spacing(3),
-  },
-}));
+const Root = styled('div')(
```

Comment:
prettier is formatting this awkward... :\

Reply:
Is it cleaner once there are more CSS properties inside? I personally find it OK-ish.

Consider the second approach works better, I think that we will need to update https://material-ui.com/guides/interoperability/#theme. Also, the first option's syntax is really hard to get right without copy & paste, so I'm happy we have the second version :)

Reply:
I was expecting the theme destructor to be on the previous line, but it's no big deal :) The second one is definitely better. 

I will go next to the guides section to see what needs to be updated 👍